internal/manifests/queryfrontend/query_frontend_test.go

```diff
@@ -76,7 +76,6 @@ func getExpectedFrontendDiscoveryService(withJaeger bool) *corev1.Service {
 			Labels:    labels,
 		},
 		Spec: corev1.ServiceSpec{
-			ClusterIP: "None",
```

Comment:
This line should not be deleted, this setting is required for the tempo cluster.